tools/walletextension/frontend/src/services/ethService.ts

```diff
@@ -14,6 +14,14 @@ const ethService = {
   checkIfMetamaskIsLoaded: async (provider: ethers.providers.Web3Provider) => {
     try {
       if (ethereum) {
+        if (ethereum.isExodus) {
+          showToast(ToastType.DESTRUCTIVE, "Exodus Wallet is connected and is conflicting with MetaMask. Please disable Exodus Wallet and try again.");
+          throw new Error("Exodus Wallet is connected and is conflicting with MetaMask. Please disable Exodus Wallet and try again.");
+        } else if (ethereum.isNestWallet) {
+          showToast(ToastType.DESTRUCTIVE, "Nest Wallet is connected and is conflicting with MetaMask. Please disable Nest Wallet and try again.");
+          throw new Error("Nest Wallet is connected and is conflicting with MetaMask. Please disable Nest Wallet and try again.");
+        }
+
```

Comment:
@zkokelj LGTM. However, is this issue isolated to these wallets only? We might want to consider creating a map that associates each wallet's name with its key. For example, `ethereum.isExodus` could correspond to 'Exodus Wallet'. Then, we can loop through the map and display the error message accordingly.